### ISSUE TITLE
Audio versions of reviews/articles

### ISSUE THREAD

**Issue by frederickobrien** [audioxide/website#84]
Originally posted in Notion 01/06/2020:

> This is waaaay down the line, but I’ve been reading about web accessibility lately and think it’s really important that the site follow best practice on that front. Most of that is just marking up content properly, but having the option of software reading our content would be pretty neat.
> 
> *The New Yorker* does something similar ([https://www.newyorker.com/magazine/2020/07/06/the-rabbit-outbreak](https://www.newyorker.com/magazine/2020/07/06/the-rabbit-outbreak)) though they’ve got enough money/patience to actually record audio versions of articles. If and when there’s sophisticated enough open source text to speech software maybe we could look at automating something similar?

@andrewbridge commented:

> Accessibility is 100% something always on my mind, and alongside structured data, I'll be adding aria tags and the like. We'll also need to make sure our colour contrast is up to scratch to really nail this as lots of sites fail to cater to visually impaired people, and all our reviews have their own colour palette. I'll create a separate issue.
> 
> That aside, spoken versions of the articles would be brilliant. We could get all techy on this and use text-to-speech services initially, going through and properly recording things after that. All sorts of companies do TTS, but one of the more natural sounding is IBMs offering https://text-to-speech-demo.ng.bluemix.net
> 
> Bear in mind also, that many visually impaired people will already have screen readers enabled.
> 
> Another thought - IF we actually did this and got the recordings done in a timely fashion, you could almost collect it together into a monthly update or podcast 😯

**Comment by frederickobrien** [comment]
https://niemanreports.org/articles/audio-articles-are-helping-news-outlets-gain-loyal-audiences/

Interesting!
